code/mod_table/mod_table.cpp

```diff
@@ -502,6 +503,12 @@ void parse_mod_table(const char *filename)
 				}
 			}
 
+			if (optional_string("$Use model eyepoint for set-camera-facing:")) {
+				stuff_boolean(&Use_model_eyepoint_for_set_camera_facing);
+				if (Use_model_eyepoint_for_set_camera_facing) {
```

Comment:
There is an opening brace at the end of this line without a closing brace.  An `if` that contains a single line doesn't need braces, but if you add them, there needs to be a pair.